src/init/pkg_installer.sh

```diff
@@ -2,52 +2,53 @@
 
 # Copyright (C) 2015-2020, Wazuh Inc.
```

Comment:
We can update the Copyright to the year in course.

Reply:
Done!